src/metabase/driver/postgres.clj

```diff
@@ -196,7 +196,7 @@
        :unix-timestamp->timestamp         unix-timestamp->timestamp
        :date                              date
        :date-interval                     date-interval
-       :set-timezone-sql                  "SET LOCAL timezone TO ?;"
+       :set-timezone-sql                  "UPDATE pg_settings SET setting = ? WHERE name ILIKE 'timezone';"
```

Comment:
This looks a little suspect to me.  Are we sure this is only setting the timezone on the connection and not trying to set the timezone for the server?